test/Servant/QuickCheck/InternalSpec.hs

```diff
@@ -119,6 +135,11 @@ type API = ReqBody '[JSON] String :> Post '[JSON] String
 api :: Proxy API
 api = Proxy
 
+type ParamsAPI = QueryParam "one" String :> QueryParam "two" String :> Get '[JSON] String
```

Comment:
Instead of `String`, I'd use `()` here. That way, every request is exactly the same, and in the test above you can just check that the query string is *exactly* `one=_&two=_` (IIRC, `_` is what `()` [de]serializes as.

Alternatively, you could use something like `Int`, so that it's easy to check in more details the form of the query string. E.g.  take 4, check that equals `one=`, drop while isNum, take 5, check *that* equals `&two=`.